Web3Swift/Transactions/EthTransactionBytes.swift

```diff
@@ -0,0 +1,153 @@
+//
+// Created by Timofey on 3/15/18.
+//
+
+import CryptoSwift
+import Foundation
+
+//Signed transaction bytes
+public final class EthTransactionBytes: BytesScalar {
+
+    private let networkID: NumberScalar
+    private let transactionsCount: NumberScalar
```

Comment:
@biboran should it be `nonce`?

Reply:
@rockfridrich lets leave it at `transactionsCount` as it makes more sense of where the nonce in the transaction is coming from. `Transaction` has `nonce` as expected.

Reply:
@biboran original idea was to add `Alice` and `Bob` private keys into Ganache initialization script. For some tests, we will need the third character who could be `Vitalik`